product_docs/docs/pgd/5/quick_start_docker.mdx

```diff
@@ -195,43 +196,118 @@ tpaexec provision democluster
 
 Since we specified Docker as the platform, TPA creates a Docker image, containers, networks, and so on.
 
-!!! SeeAlso
+!!! SeeAlso "Further reading"
     - [`tpaexec provision`](/tpa/latest/tpaexec-provision/) in the Trusted Postgres Architect documentation
 
 ### Deploying the cluster
 
 With configuration in place and infrastructure provisioned, we can now [deploy](/tpa/latest/tpaexec-deploy/) the distributed cluster.
 
 ```shell
-tpaexec deploy democluster
+tpaexec deploy democluster -e backup_wait_timeout=100
 ```
 
-TPA applies the configuration, installing the needed packages and setting up the actual EDB Postgres Distributed cluster.
+TPA applies the configuration, installing the needed packages and setting up the actual EDB Postgres Distributed cluster. The `backup_wait_timeout` setting ensures that the deployment successfully waits for the first backup to complete.
 
-!!! Tip
-    If you see a failure while processing the `barman/first_backup` task, retry this step. Depending on the speed of your test machine, 40 retries may not be enough for initialization to complete in the relevant Docker containers. 
-
-!!! SeeAlso
+!!! SeeAlso "Further reading"
     - [`tpaexec deploy`](/tpa/latest/tpaexec-deploy/) in the Trusted Postgres Architect documentation
    
-### Testing the cluster
-   
-After the successful run of the `deploy` command the cluster is ready to use. 
+## Connecting to the cluster
+
+You are now ready to log into one of the nodes of the cluster with SSH and then connect to the database. Part of the configuration process set up SSH logins for all the nodes complete with keys. To use the SSH configuration, you will need to be in the `democluster` directory created by the `tpaexec configure` command earlier.
+
+```shell
+cd democluster
+```
 
-We can now run a [test](/tpa/latest/tpaexec-test/) that ensures the cluster is running as expected:
+From there you can run `ssh -F ssh_config <hostname>` to establish an SSH connection. We will connect to `kaboom`, the first database node in the cluster:
 
 ```shell
-tpaexec test democluster
+ssh -F ssh_config kaboom
+__OUTPUT__
+[root@kaboom ~]# 
 ```
 
-## Using the cluster
+You will see that you are logged in as `root` on `kaboom` {/* admin@kaboom for AWS */}
 
-We can also do a quick hands-on test with psql. For convenience, we use the barman instance to launch psql and connect to a proxy instance. As we saw previously in config.yml, TPA has assigned the name `karakul` to the barman instance, and `kapok` to the first proxy.
+You will now need to adopt the identity of the `enterprisedb` user. This user is pre-configured and authorized to connect to the cluster's nodes.
 
 ```shell
-docker exec -it -u barman karakul psql -h kapok -p 6432 -U barman bdrdb
+sudo -iu enterprisedb
```

Comment:
I flipped this from `su` to `sudo` because the `-iu` flags only make sense for sudo; not sure if there's an advantage here either way

Reply:
Ah yes, missed that in my tidy up pass. Thanks.